agent/agents/postgres/pgstatmonitor/pgstatmonitor.go

```diff
@@ -72,6 +72,9 @@ const (
 	pgStatMonitorVersion10PG12
 	pgStatMonitorVersion10PG13
 	pgStatMonitorVersion10PG14
+	pgStatMonitorVersion11PG12
```

Comment:
new variables are not added to this function https://github.com/percona/pmm/pull/912/files#diff-ae12be77883993fc7d2b5ebc501e2fb0b0078ea1d4645c2f9327095ae0661d30R153

Reply:
Added.